### DISCUSSION TITLE
Quasar Tree Shaking with Webpack

### DISCUSSION THREAD

**Discussion by eduardojustos** [quasarframework/quasar#13179] [General - Components / Directives / etc]
I am using Vue 3 + TS + Webpack, from a standard Vue + Webpack (no Vue CLI).

I did install Quasar as the following:

Added quasar and @quasar/extra as dependencies

```
//main.ts

app.use(Quasar, {
      plugins: {
        Loading,
        Notify,
      },
      config: {
        loading: {},
        notify: {},
      },
    });
```

But the bundle got huge and the project is already far advanced. How can I tree shake it?

I know there are recommendations for using Vue CLI or Quasar CLI, but the project already existed. So, what would be the best path?

**Comment by TheDirigible** [comment]
Seriously, for Quasar docs to not even mention how to do this or if it's possible, makes me want to immediately eliminate it from the list of potential choices for a component lib.
